### PR TITLE
documentation - fix typos

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] fixes a small typo / wording issue in a docblock

### CODE REVIEW

**Comment by localheinz** [comment]
@SpacePossum 

What do you think if I just push a few more small fixes into this branch before merging? Reviewing some code, if I stumble across something, it's not really worth opening up another PR, I think.

File: src/Tokenizer/Token.php [hunk 1]
```diff
@@ -454,9 +454,9 @@ public function isMagicConstant()
     }
 
     /**
-     * Check if token is a whitespace.
+     * Check if token is whitespace.
      *
-     * @param null|string $whitespaces whitespaces characters, default is " \t\n\r\0\x0B"
+     * @param null|string $whitespaces whitespace characters, default is " \t\n\r\0\x0B"
```
Comment (keradus) [inline src/Tokenizer/Token.php]: not 2.2 ?

**Comment by keradus** [comment]
long time I did not see misc/grooming PR here ;)
Usually, back in all days, I was collecting those kind of trivials (not even minors ;P) and raised PR once per 10-15 days.
But it was old days, maybe we don't have that much minors currently to fix ;)

File: src/Tokenizer/Token.php (hunk 1 above)
Comment (localheinz) [reply src/Tokenizer/Token.php]: Adjusted!

**Comment by keradus** [comment]
Thank you @localheinz.

**Pull request closed without merging**